From 08f6312f630120e2dd9aa5d7f3a880826a5686f7 Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Thu, 30 Mar 2017 15:28:24 +0200
Subject: [PATCH 10/16] ARM64: dts: meson-gxbb-p20x: add i2s codec node

Add the node for the i2s codec Everest 7134 found on the P20x reference
design

Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi b/arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi
index 44af6eb34095..aa20b020ae89 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-gxbb-p20x.dtsi
@@ -163,6 +163,12 @@
 	cvbs_vdac_out: endpoint {
 		remote-endpoint = <&cvbs_connector_in>;
 	};
+
+	i2s_codec: external-codec {
+		#sound-dai-cells = <0>;
+		compatible = "everest,es7134";
+		status = "okay";
+	};
 };
 
 &hdmi_tx {
-- 
2.20.1

